include/alpaka/core/Config.hpp

```diff
@@ -26,7 +26,7 @@
 
 // WINDOWS
 #if !defined(ALPAKA_OS_WINDOWS)
-#    if defined(__WIN32__) || defined(__MINGW32__) || defined(WIN32)
+#    if defined(_WIN64) || defined(__MINGW64__)
```

Comment:
Why did you use `_WIN64` instead `_WIN32`. Is there a reason why we should not support 32Bit?